deep_quoridor/src/renderers/training_status.py

```diff
@@ -19,15 +19,16 @@ def end_game(self, game, result):
         if self.episode_count % self.update_every == 0:
             for agent in self.agents:
                 agent_name = agent.name()
-                avg_reward, avg_loss = agent.compute_loss_and_reward(self.update_every)
+                avg_loss, avg_reward = agent.compute_loss_and_reward(self.update_every)
+                won = result.winner == agent.name()
                 print(
-                    f"{agent_name} Episode {self.episode_count + 1}/{self.total_episodes}, Steps: {self.step}, Avg Reward: {avg_reward:.2f}, "
+                    f"{agent_name} Episode {self.episode_count + 1}/{self.total_episodes} [{'*' if won else ' '}], Steps: {self.step}, Avg Reward: {avg_reward:.2f}, "
                     f"Avg Loss: {avg_loss:.4f}, Epsilon: {agent.epsilon:.4f}"
                 )
         self.episode_count += 1
         return
 
-    def action(self, game, step, agent, action):
+    def after_action(self, game, step, agent, action):
```

Comment:
No action required, but in the future we should use the @override decorator so that there's an error if the base method is renamed